observations/README.md

```diff
@@ -0,0 +1,1279 @@
+| meta.profile\[0]                                 | fixed value = `"https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-Observation-1"`                                                                                                        |
+| identifier\[0].system                            | `"https://PSSAdaptor/{{losingOdsCode}}"` - where the `{{losingOdsCode}}` is the ODS code of the losing practice                                                                                     |
+| identifier\[0].value                             | `NarrativeStatement / id \[@root]`                                                                                                                                                                  |
+| status                                           | fixed value = `"final"`                                                                                                                                                                             |
+| code.coding\[0].system                           | fixed value = `"http://snomed.info/sct"`                                                                                                                                                            |
+| code.coding\[0].code                             | fixed value = `"37331000000100"`                                                                                                                                                                    |
+| code.coding\[0].display                          | fixed value = `"Comment note"`                                                                                                                                                                      |
+| subject                                          | reference to the mapped [Patient](../patient/README.md)                                                                                                                                             |
+| context                                          | reference to the associated [Encounter](../encounters/README.md) (if present)                                                                                                                       |
+| issued                                           | `ehrCompostion / author / time [@value]` <sup>2</sup> or else `EhrExtract / availibilityTime [@value]`                                                                                              |
+| performer\[0]                                    | Practitioner referenced in `CompoundStatement / paticipant` <sup>3</sup>                                                                                                                            |
```

Comment:
`CompoundStatement` or `NarrativeStatement` ?

Reply:
`NarrativeStatement`, I've changed it, thanks.